src/main/java/com/coveo/spillway/storage/RedisStorage.java

```diff
@@ -225,4 +135,39 @@ public void close() {
   private static final String clean(String keyComponent) {
     return keyComponent.replace(KEY_SEPARATOR, "_");
   }
+
+  public static final Builder builder() {
+    return new Builder();
+  }
+
+  public static class Builder {
+    JedisPool jedisPool;
+    String keyPrefix;
+
+    private Builder() {
+      this.keyPrefix = RedisStorage.DEFAULT_PREFIX;
+    }
+
+    public RedisStorage build() {
```

Comment:
It's silly but I would recommend that this method be moved to the bottom of the method list.
It will follow the normal "flow" of setting things and then building.

/pedant